packages/react-meteor-accounts/CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+
+Release versions follow [Semantic Versioning 2.0.0 guidelines](https://semver.org/).
+
+## v1.0.0-beta.1
```

Comment:
This needs to be updated

Reply:
Had we previously released this? I don't recall.

Reply:
All set.